lib/index.js

```diff
@@ -365,6 +343,24 @@ function stringToBuffer(string) {
 }
 
 /**
+ *
+ * Converts an ArrayBuffer to a Buffer
+ *
+ * @api private
+ */
+
+function arrayBufferToBuffer(data) {
```

Comment:
On this: with ArrayBufferView, should we only transfer the part relative to the view (like there https://github.com/websockets/ws/blob/master/lib/Sender.js#L297) or should we copy the underlying ArrayBuffer completely (as it is implemented now)?